include/onupdate.inc.php

```diff
@@ -34,6 +34,15 @@
  * function imtagging_db_upgrade_2() {
  * }
  */
+
+function imtagging_db_upgrade_2() {
+	$icmsDatabaseUpdater = icms_db_legacy_Factory::getDatabaseUpdater();
+	//$icmsDatabaseUpdater->;
+	$sql = sprintf("ALTER TABLE %s DROP column tag_cancomment", icms::$xoopsDB->prefix('imtagging_tag'));
```

Comment:
I wonder if the upgrade method handles this without having to explicitly do this. See icms_db_legacy_updater_Handler::upgradeObjectItem().

Reply:
It does - I just removed the 2 lines in class/tag.php for tag_cancomment and updated the module. That was enough to drop the column.

Reply:
Great! that makes version updates considerably easier, at least when it comes to removing fields.